security/aporeto/docs/README.md

```diff
@@ -0,0 +1,337 @@
+
+Its worth noting that each deployment config should have at least one label that uniquely identifies it. In the example below this is accomplished using the `role=api` label; no other deployment will use this specific label. See the **Naming Convention** section below for best practices. 
+
+```yaml
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    name: "${NAME}"
+    labels:
+      name: "${NAME}"
+      app: ${APP_NAME}
+      role: api
+      env: production
+    annotations:
+      description: Defines how to deploy the application server
+  spec:
+    strategy:
+  ...
+```
+
+**🤓 ProTip**
+
+* Add the `role=` label to each deployment config of your manifest to uniquely identify the pods.
+
+
+## Naming Convention
+
+The naming convention you use for labels that allow you to build your application identity should be no more complicated than necessary.
+
+| Label | Values        | Description     |
+| ----- |:-------------:|:---------------|
+| app  | name           | Use the `app` label to group all your deployment components. This *won't* be unique. |
+| role | frontend, backend, database, etc.| Use the `role` label to uniquely identify a pod by its purpose. |
+| env  | dev, test, prod | Use the `env` label to correlate with the namespace the application runs in. |
+
+In the samples below two deployments are configured in an deployment manifest. The values for `NAME` and `APP_NAME` are parameters making them common to both the Web and API deployments. This is a common convention used in OCP (and k8s) deployments.
+
+The `env` label will change based on the namespace the deployment targets and the `role` uniquely identifies the deployment.
+
+**Web**
+```yaml
+  metadata:
+    name: "${NAME}"
+    labels:
+      name: "${NAME}"
+      app: ${APP_NAME}
+      role: web
+      env: production
+```
+
+**API**
+```yaml
+  metadata:
+    name: "${NAME}"
+    labels:
+      name: "${NAME}"
+      app: ${APP_NAME}
+      role: api
+      env: production
+```
+
+**🤓 ProTip**
+
+* K.I.S - Keep It (your labels) Simple; don't try and outsmart yourself.
+* Where possible use the naming convention proposed here for uniformity across projects.
+
+## Policy
+
+The policy below will provide most teams enough to get up and running in short order; Customize them with labels or template parameters as needed. If you find the sample policy below does not suite your needs contact platform services to help create more advanced policy.
+
+In the subsections below we'll use one or more tags to identify the source and destination systems. The relevance if `source` and `destination` is that the application identified in the `source` will be able to open a network connection to the application identified in `destination`; Once a connection is open then data is able to flow bidirectionally.
```

Comment:
If the destination is protected by Aporeto as well, then a matching custom policy on that end is required in order to make the flow bidirectional.

Reply:
Yes, if they are trying to talk across namespaces then policy is needed on both ends. Within the same namespace its not required.